webapp/src/shared/components/Game/Game.tsx

```diff
@@ -85,31 +147,62 @@ const Play = ({ gameId }: GameProps) => {
   );
 };
 
-const Wait = () => {
+const ConfigureGame = ({ submitGameConfig, joinGame }: ConfigureGameProps) => {
+  const rowsData: RowItem[] = [
+    {
+      onClick: () => {
+        submitGameConfig('classic');
+        joinGame();
+      },
+      title: 'Classic game',
+      key: 'Classic game',
+      iconVariant: IconVariant.PLAY,
```

Comment:
I totally made up this icons, this should be changed but I didn't want to lose time looking for pretty ones